notebooks/101_getting-started.ipynb

```diff
@@ -111,13 +111,20 @@
    },
    "outputs": [],
    "source": [
+    "from fedbiomed.common.metrics import MetricTypes\n",
     "model_args = {}\n",
     "\n",
     "training_args = {\n",
     "    'loader_args': { 'batch_size': 48, }, \n",
     "    'optimizer_args': {\n",
     "        \"lr\" : 1e-3\n",
     "    },\n",
+    "    'test_ratio' : 0.25,\n",
+    "    'test_batch_size': 64,\n",
+    "    'test_metric': MetricTypes.F1_SCORE,\n",
+    "    'test_on_global_updates': True,\n",
+    "    'test_on_local_updates': True,\n",
+    "    'test_metric_args': {'average': 'marco'},\n",
```

Comment:
I feel like we have to keep this notebook as simple as possible. I dont think adding parameters related to validation is a good idea here